src/Core/Translation/Storage/Provider/ModuleCatalogueLayersProvider.php

```diff
@@ -154,8 +164,15 @@ public function getDefaultCatalogue(string $locale): MessageCatalogue
         // First we search in translation directory in case the module is native
         // If no translation file is found, we extract the catalogue from the module's templates
 
+        try {
+            $moduleDefaultCatalogue = $this->getBuiltInDefaultTranslatedCatalogueFinder()->getCatalogue($locale);
+        } catch (TranslationFilesNotFoundException $exception) {
+            // We ignore when module's translation directory does not exist because it's not a requirement for now
+            $moduleDefaultCatalogue = new MessageCatalogue($locale);
+        }
```

Comment:
I think this part might be tricky because the default catalogue is actually a cache of the extracted wordings. Right now it's never generated, and it also cannot be invalidated, so I wonder if it's the right time to add it.